js/app.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+function reqResource() {
+  const genReq = new XMLHttpRequest();
+  const genReqListener = function() {
+    const genObj = JSON.parse(this.responseText);
+    const contContainer = document.querySelector('#contentContainer');
+
+    while (contContainer.firstChild) {
```

Comment:
There's a much more efficient way to do this. You can utilize innerHTML to delete everything in that element in one line.